src/index.js

```diff
@@ -317,6 +321,20 @@ class SentryCliPlugin {
         return this.cli.releases.new(release);
       })
       .then(() => this.cli.releases.uploadSourceMaps(release, this.options))
+      .then(() => {
+        const { commit, previousCommit, repo, auto } = this.options;
+
+        if (repo && ((commit && previousCommit) || auto)) {
```

Comment:
`previousCommit` can be skipped in the settings, thus we only need `repo && (commit || auto)` check here.

Reply:
Fixed